arrow-optics/src/main/kotlin/arrow/optics/extensions/map/at/MapKAt.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.optics.extensions.map.at
+
+import arrow.core.MapK
+import arrow.core.Option
+import arrow.optics.PLens
+import arrow.optics.extensions.MapKAt
+
+@JvmName("at")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated(
+  "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
+  level = DeprecationLevel.WARNING
```

Comment:
All these functions are extension functions with `PLens` as the receiver which is not the same than 
```
At.map<K, V>().at(i)
```
but
```
At.map<K, V>().run { someLens.at(i) }
```
But I'm not sure if this can be expressed correctly with `ReplaceWith`.

Additionally, the typeclass has `at` extension functions for other optics like `PIso` or `PPrism` so I was wondering why only the version for `PLens` is exposed as top-level function by the `@extension` annotation.

Reply:
Oh right, I missed that. I updated the `ReplaceWith` ;) I think that actually looks better than `At.map<K, V>().run { someLens.at(i) }` 😅